Polymesh/docs/ledger.md

```diff
@@ -0,0 +1,86 @@
+
+## Using on the Polymesh App
+
+### Loading Your Account
+
+The [Polymesh App](https://app.polymesh.live/#/accounts) already has an integration with the Ledger application so that your device will work with the browser interface after installation. 
+
+The functionality is currently gated behind a feature setting that you will need to turn on.
+
+In order to turn on the interoperability with the Polymesh Ledger application, go to the "Settings" tab in the Polymesh App. Find the option for attaching Ledger devices and switch the option from the default "Do not attach Ledger devices" to "Attach Ledger via WebUSB".
+
+![Dropdown selector for allowing Ledger connections in Polymesh Settings](assets/ledger/ledger.png)
+
+Click "Save" to keep your settings.
+
+For the next step, your Ledger device must be attached to your computer, and the Ledger Live app must be closed.
+
+Now when you go to the "Accounts" tab you will see a new button that says "Add Ledger". Ensure that
+your Ledger device is unlocked and you have navigated into the Polymesh application, then click this
+button.
+
+![Add Ledger button in Polymesh](images/ledger/query-ledger.png)
+
+A popup will appear asking you to select an account and derivation path.
+
+![Picking an account and derivation path](images/ledger/add-account.png)
+
+The first option lets you select an account. You can have multiple accounts on a single Ledger device. The second dropdown lets you pick a derivation path - think of it like a formula from which child accounts are generated. If in doubt, pick the first option for both. 0 / 0 is a good default.
+
+Once you confirm your selection, depending on your browser and its security settings, you might need to confirm the USB connection through a popup like the one below when adding the Ledger device for the first time:
+
+Click on the "Unknown device" line and the "Connect" button will become available.
+
+You should now be able to scroll down and find a new account on the page with the type "ledger".
+
+You can now use this account to interact with Polymesh on Polymesh App and it will prompt your ledger for confirmation when you initiate a transaction.
+
+### Confirming the Address on your Device
+
+On the "Accounts" tab, find your Ledger-connected account. Click on the three vertical dots at the end of the row. This will open a new menu, here you can click the "Show address on hardware device" option to display the address on your device.
+
+![Options menu of an account in the Accounts screen of Polymesh](images/ledger/ledger-4.png)
+
+Here you can scroll through and make sure the address matches to what is displayed on the Polymesh App.
+
+#### Using the Polymesh App
+
+Once you have your account loaded on the "Accounts" tab it should show a row with your Ledger account. At the far right of the row is located your account's POLYX balance. If you expand the balance arrow, it will show details of your balance such as locks or reserved amounts.
```

Comment:
```suggestion
Once you have your account loaded on the *"Accounts"* tab, it should show a row with your Ledger account. At the far right of the row, your account's POLYX balance is displayed. If you expand the balance arrow, it will show details of your balance such as locks or reserved amounts.
```